apps/api/src/domain/models/transcribed-audio/entities/Transcript.ts

```diff
@@ -1,26 +1,51 @@
-import { NestedDataType } from '@coscrad/data-types';
+import { NestedDataType, NonEmptyString } from '@coscrad/data-types';
 import { DTO } from '../../../../types/DTO';
+import { AggregateId } from '../../../types/AggregateId';
 import BaseDomainModel from '../../BaseDomainModel';
-import { MediaTimeRange } from './MediaTimeRange';
+import { TranscriptItem } from './MediaTimeRange';
 
-export class Transcript extends BaseDomainModel {
-    @NestedDataType(MediaTimeRange, {
+class TranscriptParticipant {
+    @NonEmptyString({
+        label: 'speaker label',
+        description: 'a label (e.g., initials) for this speaker',
+    })
+    readonly label: string;
+
+    // This should eventually point to a "Person" model
```

Comment:
Are you thinking about this as part of our first pass?  Or will this take more work than we should commit right now?

Reply:
Right now and first pass are 2 different timelines. Right now is this PR. It's outside the scope of this PR. On the other hand, we can talk about where it sits as far as priorities for the MVP. I think it's worth doing once we support context on the frontend and we support command flows for all existing models.